Makefile

```diff
@@ -136,6 +166,12 @@ cni-plugins: get-cni-sources
 run-integ-tests: test-registry gremlin
 	. ./scripts/shared_env && go test -race -tags integration -timeout=5m -v ./agent/engine/... ./agent/stats/... ./agent/app/...
 
+codebuild: out get-deps test-artifacts-in-docker
+	$(MAKE) release TARGET_OS="linux"
```

Comment:
These target names (`release`, `docker-release`) seem a little backwards to me after reading them, `release` depends on `docker-release` but `release` is actually the target that bundles the agent into a Docker container image (according to the comments). Likely a conversation for another PR.

Reply:
Yep. Theres a lot of simplification and cleanup we could do -- but for now I was only trying to mirror what the staging and building scripts do.